.github/workflows/release.yml

```diff
@@ -0,0 +1,94 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 12.x
+
+      - name: Install Dependencies
```

Comment:
Recommend using `actions/cache@v2` here to reduce the run time of the average build, if dependencies haven't changed. I've gone off of [this article's comparisons](https://www.voorhoede.nl/en/blog/super-fast-npm-install-on-github-actions/) before in terms of which caching strategy is best, but any of them would help reduce the number of GH action minutes each run takes

Reply:
Partially resolved via 3e302768399155f7c1a723693404188a4b2a477f & 58875fcbceeff7fdf9dbca6a98c456753ca24af4 as `actions/setup-nodev3` includes caching support out of the box, so this just needs to be configured rather than adding `actions/cache`

https://github.com/actions/setup-node#caching-global-packages-data